### PR TITLE
events: page-update

### PR DESCRIPTION
**Pull request by Stanislaw-bnk**

### CODE REVIEW

File: content/events.md [hunk 1]
```diff
@@ -73,16 +66,25 @@ all_events:
 
   - year: "20**24**"
     events:
+    - title: "Dasharo User Group 0x5 **& Developers vPub 0xA**"
+      status: "upcoming"
```
Comment (artur-rs) [inline content/events.md:70]: @Stanislaw-bnk wrong status

File: content/events.md [hunk 2]
```diff
@@ -210,12 +241,12 @@ all_events:
             * [**Slides**](https://dl.3mdeb.com/dasharo/dug/2/)
 
 
-    - title: "Yocto Project at **Embedded Open-Source Summit**"
+    - title: "Yocto Project **Dev Days**"
```
Comment (artur-rs) [inline content/events.md:244]: @Stanislaw-bnk shouldn't we leave the historical events' names?

**Review by artur-rs — CHANGES_REQUESTED** [review]

File: content/events.md (hunk 2 above)
Comment (Stanislaw-bnk) [reply content/events.md:244]: This change is based on this comment: https://chat.3mdeb.com/team-3mdeb/pl/cnh6ktq38b8jujx3u4kppujfxw

Comment (pietrushnic) [reply content/events.md:244]: @Stanislaw-bnk, this is a very bad idea to link private discussions from our company chat to a public repository; please remove that.

Comment (artur-rs) [reply content/events.md:244]: Thanks for clarification

**Review by artur-rs — APPROVED** [review]

**Pull request merged**